src/EconomicStrategy/IEconomicStrategy.ts

```diff
@@ -33,4 +33,11 @@ export interface IEconomicStrategy {
   minExecutionWindow?: number;
 
   minExecutionWindowBlock?: number;
+
+  /**
+   * Smart gas estimation will use the Eth Gas Station API to
+   * retreive information about the speed of gas prices and pick
+   * the gas price which better fits execution situations.
+   */
+  usingSmartGasEstimation?: boolean;
```

Comment:
This does not have to be nullable, default will be false. Or we can set it to true by default.

Reply:
Some tests assume all fields on `IEconomicStrategy` are nullable.

Reply:
`Config` sets to `false` as default but that can be overwritten by passing it as an option when creating a new `Config` from the `eth-alarm-clock-app` or the `cli`.